samples/unit-tests/maps/mapbubble-linewidth/demo.js

```diff
@@ -51,4 +51,10 @@ QUnit.test('MapBubble', function (assert) {
         `When hovering over mapbubble series with shared tooltip,
         there should be no errors in the console.`
     );
+
+    // Passing the default joinBy value for MapBubbleSeries (#16750).
+    assert.ok(
+        chart.series[1].joinBy[0] === "hc-key",
+        `Checking default options for joinBy: 'hc-key'.`
```

Comment:
One of our internal rules, when writing unit tests, is that description of a test should have a "should" keyword.

For example:
```
The value of joinBy in the mapbubble series should be 'hc-key' by default.
```